.ci/test-project/packages/uberenv-magictestlib/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
```

Comment:
Is this one for compatibility with the older workflows?

Reply:
this test is based on how i use uberenv across projects, but we can also add things that test the dev build but I am not the expert on that way of using.